README.md

```diff
@@ -204,6 +204,11 @@ The low-level API is as follows:
 
 For examples how to use them, please refer to the README of the above mentioned solc-js releases.
 
+**Note**: These low-level functions remain available for compatibility reasons.
+However, they were superseded by the `compile()` function and are no longer required.
+Starting in version `0.5.0+commit.1d4f565a`, the functions `compileSingle`, `compileMulti`, and `compileCallback` are always `null` when using newer binary versions.
```

Comment:
```suggestion
Starting from version `0.5.0+commit.1d4f565a`, the functions `compileSingle`, `compileMulti`, and `compileCallback` are always `null` when using newer binary versions.
```
Also, I find the last part, `when using newer binary versions`, a little bit confusing because it doesn't specify which versions are `newer` and that those are `solc` binaries, not `solc-js` (right?).

Reply:
Right. Fixed.